### PR TITLE
LIA-64235 add support for CURSOR command, upgrade guava dependency.

### PR DESCRIPTION
**Pull request by dougschroeder**

### CODE REVIEW

File: src/test/java/com/lithium/ldn/starql/parsers/JparsecPageConstraintsTest.java [hunk 1]
```diff
@@ -45,4 +45,28 @@ public final void test_pageConstraints4() {
 		assertEquals(defaultPage,
 				inst.pageConstraintParser().parse("limit 10 offset 100"));
 	}
+
+	@Test
+	public final void test_pageConstraints5() {
+		assertEquals(defaultLimitAndCursor,
+				inst.pageConstraintParser().parse("LIMIT 10 CURSOR 'eyJsaWFWZXJzaW9uIjoiMTkuOCAoYWN0aXZlKSIsImN1cnNvclR5cGUiOiJxdWVyeSJ9'"));
```
Comment (YuriKap) [inline src/test/java/com/lithium/ldn/starql/parsers/JparsecPageConstraintsTest.java]: The same cursor string is used in multiple places, probably worthwhile to pull it out as a constant.

File: pom.xml [hunk 2]
```diff
@@ -26,7 +31,7 @@
 		<dependency>
 			<groupId>com.google.guava</groupId>
 			<artifactId>guava</artifactId>
-			<version>14.0.1</version>
+			<version>27.1-jre</version>
```
Comment (li-a) [inline pom.xml]: Thanks

File: pom.xml [hunk 3]
```diff
@@ -26,7 +31,7 @@
...
 			<optional>true</optional>
```
Comment (li-a) [inline pom.xml]: Not optional

File: src/main/java/com/lithium/ldn/starql/models/QlPageConstraints.java [hunk 4]
```diff
@@ -12,51 +11,67 @@
...
 
 	@Override
 	public String toString() {
-		return "PageConstraints [limit=" + limit + ", offset=" + offset + "]";
+		return "PageConstraints [limit=" + limit + ", offset=" + offset + ", cursor=" + cursor + "]";
 	}
 
 	@Override
-	public int hashCode() {
-		return HashCodeBuilder.reflectionHashCode(this);
```
Comment (li-a) [inline src/main/java/com/lithium/ldn/starql/models/QlPageConstraints.java]: Great

**Pull request merged**